sonarts-sq-plugin/sonar-typescript-plugin/src/test/java/org/sonar/plugin/typescript/ExternalTypescriptSensorTest.java

```diff
@@ -237,7 +237,9 @@ public void should_log_when_tsconfig_for_file_not_found() throws Exception {
 
     executeSensor(sensorContext, testBundle);
 
-    assertThat(logTester.logs()).contains("No tsconfig.json file found for " + inputFile.uri() + " (looking up the directories tree). This file will not be analyzed.");
+    assertThat(logTester.logs()).contains("No tsconfig.json file found for [" +
+      inputFile.uri() + "] (looking up the directories tree until project base directory [" +
```

Comment:
I think that test should not use `inputFile.uri()` but actual URI as string literal

Reply:
@saberduck indeed! fixed in 42b4ed1